### PR TITLE
Add general static asset handling and badges to Challenge

### PR DESCRIPTION
**Pull request by nikriek**

### CODE REVIEW

**Comment by nicoknoll** [comment]
Tests failing...

File: challenges/tests/test_urls.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
 from django.test import TestCase
 from django.core.urlresolvers import reverse
+from challenges.models import CHALLENGES
```
Comment (nicoknoll) [inline challenges/tests/test_urls.py]: Why uppercase?

**Review by nicoknoll — APPROVED** [review]

File: challenges/tests/test_urls.py (hunk 1 above)
Comment (nikriek) [reply challenges/tests/test_urls.py]: Because it's a constant

**Comment by nikriek** [comment]
It's Work In Progress

**Pull request merged**